README.md

```diff
@@ -39,28 +39,28 @@ assert value == "hello world!"
 
 ## Development Setup
 
-0. Install Rust and Python
-1. Install `maturin` in order to build Ypy
+0. [Install Rust](https://www.rust-lang.org/tools/install) and [Python](https://www.python.org/downloads/) 
+  - Some alternative bootstrapping options are [pyenv](https://github.com/pyenv/pyenv) for Python, or using [mamba](https://github.com/conda-forge/miniforge) to `mamba install rust python`.
+1. Install [hatch](https://hatch.pypa.io/latest/install/) (`python -m pip install hatch`)
+2. Create a development build of the library
```

Comment:
Maybe we should remain agnostic to the users' package manager of choice in the setup. If we provide the default instructions with `pip` and have a `pyproject.toml`, more advanced users will be able to leverage it regardless of whether they are using poetry or hatch

Reply:
I'm good with whatever wording you want to use here. I think the original docs got across the point that you need bootstrapping before being able to build ypy, and these do as well.

Reply:
I'm not entirely sure what tweaks you're asking for but I'm 👍 if you want to commit directly to the PR with the wording you'd like to see here. I think this may be the last item to resolve in the PR?